P/protoc/build_tarballs.jl

```diff
@@ -34,18 +19,35 @@ cd $WORKSPACE/srcdir/protobuf
 sed -i -e 's!set(CMAKE_C_COMPILER.*!set(CMAKE_C_COMPILER '${WORKSPACE}/srcdir/files/ccsafe')!' ${CMAKE_TARGET_TOOLCHAIN}
 sed -i -e 's!set(CMAKE_CXX_COMPILER.*!set(CMAKE_CXX_COMPILER '${WORKSPACE}/srcdir/files/c++safe')!' ${CMAKE_TARGET_TOOLCHAIN}
 
-git submodule update --init --recursive
-cmake -B build -G Ninja \
+# Build static libraries
```

Comment:
Probably better to split package into separate lib, lib-static and protoc packages.